app/controllers/chef_profiles_controller.rb

```diff
@@ -1,7 +1,23 @@
 class ChefProfilesController < ApplicationController
+  before_action :skip_authorization
   def show
+    @chef_profile = ChefProfile.find(params[:id])
   end
 
   def create
+    @chef_profile = ChefProfile.new(chef_params)
+    @chef_profile.user = current_user
+    # We run the authorize just before saving
+    # so that the instance is fully set
+    authorize @chef_profile
+    if @chef_profile.save
+      redirect_to @chef_profile, notice: 'Restaurant was successfully created.'
```

Comment:
This message here might need to change because we creating a chef profile, not a restaurant. I would suggest editing this message before merging this branch with this master, so we don't forget to edit it in the future.